source/_integrations/flexmeasures.markdown

```diff
@@ -0,0 +1,175 @@
+---
+title: FlexMeasures
+description: Use FlexMeasures flexible energy asset in Home Assistant 
+ha_category:
+  - Energy
+ha_release: "2023.9"
+ha_iot_class: Cloud Polling
+ha_codeowners:
+  - '@victorgarcia98'
+  - '@GustaafL'
+  - '@Flix6x'
+ha_domain: flexmeasures
+ha_platforms:
+  - sensor
+ha_integration_type: integration
+ha_config_flow: true
+ha_quality_scale: silver
+---
+
+
+
+
+The `flexmeasures` integration offers an integration with [FlexMeasures](https://flexmeasures.io/) instances to schedule flexible energy assets.
+
+FlexMeasures can be used to schedule batteries, EV's, heat storage, and other assets. The current integration only support car batteries and batteries optimizations. Shiftable energy assets like dishwashing machines are currently not supported in the integration. The integration offers the following sensor and services:
```

Comment:
```suggestion
FlexMeasures can be used to schedule batteries, EV's, heat storage, and other assets. The current integration only supports car batteries and batteries optimizations. Shiftable energy assets like dishwashing machines are currently not supported in the integration. The integration offers the following sensor and services:
```

Reply:
Thank you for reviewing and finding these errors! Commited your suggestions.